pkg/haprovider/haprovider.go

```diff
@@ -270,11 +272,11 @@ func (r *HAProvider) syncEndpointMachineIP(address corev1.EndpointAddress, machi
 			if machineAddress.Type != clusterv1.MachineExternalIP {
 				continue
 			}
-			if net.ParseIP(machineAddress.Address) != nil && net.ParseIP(machineAddress.Address).To4() != nil {
```

Comment:
Remove ipv4 check here, in this way we allow IPv6 type of control plane endpoint from akoo side.
But actually we can't support IPv6 control plane endpoint because of avi controller limitation. Prefer to add the dataplane network/controlplane network check in tkg cli side, so that when avi controller limitation is resolved, ako-operaoter can directly support IPv6 control plane endpoint.